pandas/tests/scalar/timestamp/test_timezones.py

```diff
@@ -26,14 +26,14 @@ def test_tz_localize_pushes_out_of_bounds(self):
         pac = Timestamp.min.tz_localize("US/Pacific")
         assert pac.value > Timestamp.min.value
         pac.tz_convert("Asia/Tokyo")  # tz_convert doesn't change value
-        with pytest.raises(OutOfBoundsDatetime):
+        with pytest.raises(OutOfBoundsDatetime, match="^$"):
```

Comment:
would it make sense to have a message here?

Reply:
the current implementation returns nothing, it's indeed should better have some message but that's out of the issue's scope

Reply:
yep, out of scope. but if you could raise an issue, that'll be great.